OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -643,7 +653,8 @@ public void TestInvalidDispatchImpressionEvent()
             LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, "Assigned bucket [3037] to user [test_user] with bucketing ID [test_user]."), Times.Once);
             LoggerMock.Verify(l => l.Log(LogLevel.INFO, "User [test_user] is in variation [control] of experiment [test_experiment]."), Times.Once);
             LoggerMock.Verify(l => l.Log(LogLevel.INFO, "Activating user test_user in experiment test_experiment."), Times.Once);
-            LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, "Dispatching impression event to URL OptimizelySDK.Event.LogEvent with params {\"param1\":\"val1\"}."), Times.Once);
+            LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, "Dispatching impression event to URL logx.optimizely.com/decision with params {\"param1\":\"val1\"}."), Times.Once);
```

Comment:
bhai jaan use $ where possilbe :p

Reply:
No need of $ here as it is used for variables and expressions. Using @ and replacing \" with "".